src/pokemon/pokeitem.js

```diff
@@ -1,16 +1,19 @@
 import React, { Component } from 'react'
+import { Link } from 'react-router-dom';
 
 export default class PokeItem extends Component {
     render() {
         return (
-            <li className="pokeItems">
-                <p className="title">{this.props.pokemomItemProp.pokemon}</p>
-                <img src={this.props.pokemomItemProp.url_image} alt={this.props.pokemomItemProp.pokemon}></img>
-                <p>Type: {this.props.pokemomItemProp.type_1}</p>
-                <p>HP: {this.props.pokemomItemProp.hp}</p>
-                <p>Attack: {this.props.pokemomItemProp.attack}</p>
-                <p>Defense: {this.props.pokemomItemProp.defense}</p>
-            </li>
+            <Link to={this.props.pokemomItemProp.pokemon} className="poke-display">
```

Comment:
This looks good! To see the pokemon details you will need to add a Route in App.js that renders the DetailPage component :)